_pages/about-us/teams/products-and-platforms.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: Products and Platforms
 ---
-Products and Platforms is one of the 5 business units within 18F. We strive to incubate tools for digital government.
```

Comment:
How about:  "We initiate the development of tools for digital government." or "We initiate the development of digital tools for government."